index.js

```diff
@@ -9,7 +9,6 @@ module.exports = {
       exports: 'always-multiline',
       functions: 'always-multiline',
     }],
-    'object-curly-spacing': ['error', 'always'],
```

Comment:
This was remove, because this is now the default value in eslint-config-standard v12.0.0